testdata/scripts/seed.txt

```diff
@@ -0,0 +1,36 @@
+# Check the binary with a given base64 encoded seed
+garble -debugdir=./debug -literals -seed=OQg9kACEECQ= build main.go
```

Comment:
please don't mix other features like -literals into this test - you should be able to test the obfuscation of identifiers, which always happens.

Reply:
The feature also affects the strings generated by -literals

Reply:
Ah, good point.